run.py

```diff
@@ -0,0 +1,8 @@
+from tutor import app, db
+
+
+from tutor.routing import *
+
+
+if __name__ == "__main__":
```

Comment:
I don't expect to see this in a Flask app since those are not typically supposed to be invoked on  their own.

I suspect that just importing `app` from `tutor.routing` would be enough.

Reply:
The routing system is configured differently on the next PR. I focused on that PR on DB initialization.